internal/api/v1/service/create.go

```diff
@@ -61,8 +62,25 @@ func Create(c *gin.Context) apierror.APIErrors {
 		return apierror.InternalError(err)
 	}
 
+	// Validate the chart values, if any.
+	if len(createRequest.Settings) > 0 {
+		issues := application.ValidateCV(createRequest.Settings, catalogService.Settings)
+		if issues != nil {
+			// Treating all validation failures as internal errors.
```

Comment:
Those are treated as bad request (that looks fine to me). Wrong/old comment?

Reply:
Likely a copy/paste error. And possibly wrong in the origin location also.